tools/tox-lxd-runner

```diff
@@ -78,7 +79,7 @@ lxc file push --recursive --create-dirs "$uapath" "$container/$(dirname "$c_uapa
 echo "[RUNNING TESTS]"
 
 testrun_rc=1
-lxc exec "$container" --cwd "$c_uapath" -- "$@" && testrun_rc=0 || testrun_rc=$?
```

Comment:
Why did you remove the `&& testrun_rc=0` ? The tests are passing but finishing as failures because of that modification

Reply:
@lucasmoura it was not intended, I updated the PR.